editoast/src/views/train_schedule/mod.rs

```diff
@@ -102,17 +106,17 @@ async fn delete(db_pool: Data<DbPool>, train_schedule_id: Path<i64>) -> Result<H
 #[patch("")]
 async fn patch(
     db_pool: Data<DbPool>,
-    train_schedules_changeset: Json<Vec<TrainScheduleChangeset>>,
+    train_schedules_changesets: Json<Vec<TrainScheduleChangeset>>,
     core: Data<CoreClient>,
 ) -> Result<HttpResponse> {
-    let train_schedules_changeset = train_schedules_changeset.into_inner();
-    if train_schedules_changeset.is_empty() {
+    let train_schedules_changesets = train_schedules_changesets.into_inner();
+    if train_schedules_changesets.is_empty() {
```

Comment:
I'd argue that an empty list of changeset should return a 204.

Reply:
204 is used if the request has succeeded, which is not the case here.

Reply:
There's no modification to apply, so by doing nothing you comply to what the request asked. The user asked you to change nothing, you changed nothing, success!